introduction.tex

```diff
@@ -62,6 +62,9 @@ \subsection{Register Definition Format}
         \hline
         R/W1 & Read/Write. Only writing 1 has an effect.  \\
         \hline
+        R/W1C & Read/Write. Writing every bit high clears the field. Writing
```

Comment:
I think we're not talking about exactly the same thing. Let's take cmderr, which is currently a 3-bit field. It can contain 0 (no error), as well as several other values for error codes (1, 2, 3, 4, 7).
An alternate encoding would be to use 5 bits, one for each of the possible errors. I had thought that is what you were talking about, but now it sounds like that's not the case. I am not proposing that.

So let's stick with the 3-bit encoding. What we're discussing is how these errors should be cleared. Let's say the error code is 2. There are 3 options I can think of:
1. Write 7 (every bit high)
2. Write 2 (only the bits that are set in the error)
3. Write 0 (the value that you want to see in the register)

3 was discarded because it's too easy to accidentally clear an error.
I don't like 2 because of the race I described, but I don't know of an actual case where it's a problem since all (?) our error codes are sticky.
1 is my preferred solution.

But I don't feel very strongly about that. I just think it needs to be documented, which was the whole point of this change. :-)